### PR TITLE
feat: new pricing page with updated plans

### PR DESCRIPTION
**Pull request by timmywil**
### PR Checklist

Please review the [guidelines for contributing](CONTRIBUTING.md) to this repository.

- [x] I am requesting to **pull a topic/feature/bugfix branch** (right side). In other words, not _master_.
- [x] I have run `npm test` against my changes and tests pass.
- [x] I have added or edited necessary documentation, or no docs changes are needed.

### Description

This implements the new design of the pricing page.

Remaining items:

- [x] FAQ section
- [x] Cross-browser testing
- [x] Noel to fill out FAQ content

### CODE REVIEW

**Comment by timmywil** [comment]
@noelweichbrodt The FAQ section needs content. The questions in the design are a bit different than the ones in notion, but neither look complete. I leave it to you to decide what questions to put here.

**Comment by noelweichbrodt** [comment]
yeah, that's totally fine, actual q&as are wip

File: src/components/BlogPost.tsx [hunk 1]
```diff
@@ -55,20 +54,18 @@ export default function BlogPost({ location, post, related }: Props) {
               tags={post.fields.tags?.filter(Boolean) as string[]}
               header="Related Tags"
             />
-            {related && !!related.length && (
-              <Fragment>
+            {!!related?.length && (
```
Comment (space-pope) [inline src/components/BlogPost.tsx:58]: Doesn't this simplify to `related?.length`?

File: src/components/pricing/FAQ.tsx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        <p className="title">
+          Need more help? Check out our <a href="/docs">documentation</a>,{' '}
+          <a href="https://forums.spokestack.io">forum</a>, or{' '}
+          <a href={`mailto:${contact.email}`}>email us</a>.
+        </p>
+      </div>
+      <div css={styles.questions}>
+        <Question
+          question="What is the starting cost of Spokestack Enterprise?"
+          answer="Spokestack Enterprise is priced according to your custom requirements, designed in consultation with your company's unique business needs."
```
Comment (space-pope) [inline src/components/pricing/FAQ.tsx:25]:
```suggestion
          answer="Spokestack Enterprise is priced according to your custom requirements, designed in consultation with your company."
```

File: src/components/pricing/FAQ.tsx [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+        />
+        <Question
+          question="Does Spokestack Enterprise charge for cloud requests?"
+          answer="Spokestack Entrprise pricing is tailored to your precise use case--unmetered cloud requests included."
```
Comment (space-pope) [inline src/components/pricing/FAQ.tsx:29]:
```suggestion
          answer="No. Spokestack Enterprise pricing is tailored to your use case—unmetered cloud requests included."
```

File: src/components/pricing/FAQ.tsx [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+          question="How can I use Spokestack?"
+          answer={
+            <p>
+              Spokestack offers low-code integrations for many other voice
+              technology providers, as well as offering our own{' '}
+              <a href="https://spokestack.io/docs/node">Node</a>,{' '}
+              <a href="https://spokestack.io/docs/react-native">React Native</a>
+              , <a href="https://spokestack.io/docs/ios">iOS</a>,{' '}
+              <a href="https://spokestack.io/docs/android">Android</a>, and{' '}
+              <a href="https://spokestack.io/docs/python">Python</a> platform
```
Comment (space-pope) [inline src/components/pricing/FAQ.tsx:41]:
```suggestion
              <a href="/docs/node">Node</a>,{' '}
              <a href="/docs/react-native">React Native</a>
              , <a href="/docs/ios">iOS</a>,{' '}
              <a href="/docs/android">Android</a>, and{' '}
              <a href="/docs/python">Python</a> platform
```

File: src/components/pricing/FAQ.tsx [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+        <Question
+          question="Does Spokestack Enterprise charge for cloud requests?"
+          answer="Spokestack Entrprise pricing is tailored to your precise use case--unmetered cloud requests included."
+        />
+        <Question
+          question="How can I use Spokestack?"
+          answer={
+            <p>
+              Spokestack offers low-code integrations for many other voice
+              technology providers, as well as offering our own{' '}
```
Comment (space-pope) [inline src/components/pricing/FAQ.tsx:36]:
```suggestion
              technology providers as part of our{' '}
```

File: src/components/pricing/FAQ.tsx [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+          question="What free universal models are available?"
+          answer="We offer our highest quality TTS male voice of a Hollywood voice actor in a professional recording studio for free. In addition, we offer a free universal 'Spokestack' wake word model, digits and commands keyword models, and a trivia, Minecraft recipe, and high-low NLU models."
+        />
+        <Question
+          question="What happens when I hit my usage limits?"
+          answer="There are no penalties associated with reaching your cloud API usage limits. We will contact you to determine the best plan for your cloud API usage needs."
+        />
+        <Question
+          question="What are my payment options?"
+          answer="Spokestack accepts payment from all major credit cards and  with Google Pay."
```
Comment (space-pope) [inline src/components/pricing/FAQ.tsx:56]:
```suggestion
          answer="Spokestack accepts payment from all major credit cards and Google Pay."
```

File: src/components/pricing/FAQ.tsx [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+              our offerings very affordable in an effort to speed the adoption
+              of voice. In fact, if you compare the cost of creating a custom
+              wake word with Spokestack to that of making one with one of our
+              competitors with similar performance, you&aposll find that our
+              price is still rather low. Making a wake word or keyword spotting
+              model that performs well for a wide variety of people requires
+              collecting a substantial amount of data; it&aposs not something
+              that you can just generate. We take care of that data collection
+              on your behalf, and we&aposll notify you when the task is
+              complete, and your model is ready to train. If you have a
```
Comment (space-pope) [inline src/components/pricing/FAQ.tsx:73]:
```suggestion
              complete, meaning your model is ready to train. If you have a
```

File: src/components/pricing/OpenSource.tsx [hunk 8]
```diff
@@ -0,0 +1,190 @@
...
+            <li>Web</li>
+            <li>Python</li>
+          </ul>
+        </OpenSourceFeature>
+        <OpenSourceFeature
+          title="Drop-In Interface Widget"
+          description={
+            <p>
+              Get a head start on your projects with{' '}
+              <a href="/docs/concepts/tray">Spokestack Tray</a> available for:
```
Comment (space-pope) [inline src/components/pricing/OpenSource.tsx:76]:
```suggestion
              <a href="/docs/concepts/tray">Spokestack Tray</a>, available for:
```

File: src/components/pricing/OpenSource.tsx [hunk 9]
```diff
@@ -0,0 +1,190 @@
...
+            <li>Dialogflow</li>
+            <li>Rasa</li>
+          </ul>
+        </OpenSourceFeature>
+        <OpenSourceFeature
+          title="Flexibility to Choose TTS AI Voices"
+          description="Use the default AI voice provided by default or use an AI voice from one of the following providers*:">
+          <ul>
+            <li>Apple</li>
+            <li>Android</li>
```
Comment (space-pope) [inline src/components/pricing/OpenSource.tsx:122]:
```suggestion
            <li>Google</li>
```

File: src/components/pricing/OpenSource.tsx [hunk 10]
```diff
@@ -0,0 +1,190 @@
...
+              </a>
+            </li>
+          </ul>
+        </OpenSourceFeature>
+        <OpenSourceFeature
+          title="3rd-Party Provider Support for ASR"
+          description="Use Spokestack ASR or choose from one of the following providers*:">
+          <ul>
+            <li>Apple</li>
+            <li>Android</li>
```
Comment (space-pope) [inline src/components/pricing/OpenSource.tsx:102]:
```suggestion
            <li>Apple (on-device)</li>
            <li>Google (on-device)</li>
```

File: src/components/pricing/OpenSource.tsx [hunk 11]
```diff
@@ -0,0 +1,190 @@
...
+            <li>Microsoft</li>
+            <li>Google</li>
```
Comment (space-pope) [inline src/components/pricing/OpenSource.tsx:104]:
```suggestion
            <li>Google Cloud Speech</li>
```

**Review by space-pope — COMMENTED** [review]
Just had a couple comments/suggestions for the text.

File: src/components/BlogPost.tsx (hunk 1 above)
Comment (timmywil) [reply src/components/BlogPost.tsx:58]: That's fine most of the time but in this case it would render "0".

**Comment by timmywil** [comment]
Sorry should have waited for content review. I can add these tho.

**Comment by noelweichbrodt** [comment]
All the suggestions LGTM

**Comment by timmywil** [comment]
Added e7c5f2c

**Pull request merged**